Task70/Task70/App/Application.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+using Task70.Model;
+using Task70.Pages;
+
+namespace Task70.App
+{
+    public class Application
+    {
+        private readonly IWebDriver _driver;
+
+        private readonly HomePage homePage;
+        private readonly LoginPage loginPage;
+
+        public Application()
+        {
+            _driver = new ChromeDriver();
+            homePage = new HomePage(_driver);
+            loginPage = new LoginPage(_driver);
+        }
+
+        public void LoginUser(User user)
+        {
+            homePage.Open();//нужен ли нам вообще он тут??
```

Comment:
It is better to exclude this method because there is a chance that you will need to open browser with a different URL